cmake/Dependencies.cmake

```diff
@@ -1247,6 +1247,19 @@ endif()
 
 # ---[ HIP
 if(USE_ROCM)
+  # This prevents linking in the libtinfo from /opt/conda/lib which conflicts with ROCm libtinfo.
+  # Currently only active for Ubuntu 20.04 and greater versions.
+  if(UNIX)
+    execute_process(COMMAND bash -c "cat /etc/os-release | grep VERSION_ID | awk -F\\\" '{print $2}'" OUTPUT_VARIABLE OS_VERSION)
```

Comment:
@estewart08 @pruthvistony I think we should be checking for the NAME field to be "Ubuntu" as well. Otherwise, we'll be comparing CentOS VERSION_ID eg. "7" to "20.04", which works just because of the way the numbers line up, but seems wrong to compare them.